tests/conversion

```diff
@@ -272,6 +282,41 @@ if hasNeededAPIExtension instance_import_conversion; then
     conversion_vm vm-u24-vmdk-dir-bupvol dir "${IMAGE_PATH}" "yes"
     conversion_vm vm-u24-vmdk-zfs-bupvol zfs "${IMAGE_PATH}" "yes"
 
+    # Test virtio conversion if virt-v2v-in-place is installed and meets the minimum version
+    # requirements. Version will be satisfied from Ubuntu 24.04 runners onwards.
+    v2vMinVersion="2.3.4"
```

Comment:
While this command/version detection logic seems sound, I worry that it's a bit brittle and we risk skipping the v2v conversion if it doesn't work.

Reply:
With your suggestion above to check the OS version instead of virt-v2v version, this is no longer required. If the installed version is for any reason lower then 2.3.4, LXD will throw an intuitive error, so it should be easy to catch.